entrypoint.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
 set -e
 
-# start server
-python manage.py migrate && python manage.py runserver 0.0.0.0:${APP_PORT}
+# run migrations
+python manage.py migrate
 
+if [ ${APP_ENV} = "local" ]; then
+    python manage.py loaddata organizations/fixtures/default_tenant.yaml
+fi
```

Comment:
wouldn't we have to set up the public schema for staging and prod too?

Reply:
yea we'll have to, I'm just thinking how that can happen. Given we only need public schema once and if we add a fixture with hardcoded domain, it only makes sense for Plio. Anyone else cloning this repo and setting up for themselves won't need those specific configs.

Let me work on this separately. #140